### DISCUSSION TITLE
Just another thanx !!!

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#4187] [General]
Hi developers,

enclosed another huge THANX for your fantatstic work!!! Especially after the last update with huge improvements for the autopilot and engine model!!! I love it! Rock´n roll !!! 

greets Klaus

**Comment by rpoulachon** [comment]
Hi,

Totally agree with you, a wonderfull work made by fantastic people.

I am waiting for your last changes every day.

Best regards.
